### ISSUE TITLE
How to change background image and color

### ISSUE THREAD

**Issue by eugenevk** [JanStevens/angular-growl-2#128]
Hi, I want the icon to show in white, but can't succeed in doing so. E.g. error shows a red icon (see attached) 
Can you please advise how I can accomplish that?
Ideally I can use a custom FontAwesome icon. Is that possible?
![growl error alert](https://user-images.githubusercontent.com/19785727/36718844-7a06617a-1ba3-11e8-879a-cfea13237185.JPG)

**Comment by royerboat** [comment]
Yes, overriding the default icons with FontAwesome is possible. You can do this in a few ways, but the simplest is probably just overriding the default template with a custom one where you can use the fontawesome icons.

See https://fritz.ninja/angular-growl-2/#customization for details.

Just copy the existing template from the project, put it in a script block, and add the icons that you want to appear. You might also have to override some the CSS.

Alternatively, just override the CSS for those icons with the corresponding FontAwesome icon that you want.

**Comment by vik-singh** [comment]
i replied
